csfieldguide/static/interactives/city-trip/js/city-trip.js

```diff
@@ -0,0 +1,605 @@
+const cytoscape = require('cytoscape');
```

Comment:
You should be able to resume path finding from where it is after the algorithm is stopped, or at least reset to the same graph

Reply:
Might add this as an issue as it will take a bit of time to implement and would rather get the chapter out